### PR TITLE
Gift cards: Improved support for cross-organizer acceptance

### PR DESCRIPTION
**Pull request by raphaelm**

### CODE REVIEW

**Comment by pc-coholic** [comment]
rebased and updated migration

**Comment by pc-coholic** [comment]
I am seeing one issue, which could however also be a non-issue: Duplicate UIDs.

If I enroll a medium in two separate organizers and do not enable cross-acceptance, things will be alright.

However: if I do enable cross-acceptance, the medium of the currently active organizer/event-owner will be given priority. Even if that medium has been disabled.

I could see this as a "works as intended", but it could cause issues when organizers allow to use foreign mediums and then later decide to allow proper cross-acceptance. Or use very simple barcodes.

**Comment by pc-coholic** [comment]
The POS-transaction view does only list the giftcard (including a link to it) as the payment method. But clicking the link will yield a 401, due to it belonging to a different organizer.

File: src/pretix/control/templates/pretixcontrol/organizers/giftcard_acceptance_list.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        {% else %}
+            <div class="table-responsive">
+                <table class="table table-condensed table-hover">
+                    <thead>
+                    <tr>
+                        <th>{% trans "Organizer" %}</th>
+                        <th>{% trans "Status" %}</th>
+                        <th></th>
+                    </tr>
+                    </thead>
```
[the remark this answers is not in the document]
Comment (raphaelm) [reply src/pretix/control/templates/pretixcontrol/organizers/giftcard_acceptance_list.html:37]: :+1: will add

File: src/pretix/control/navigation.py [hunk 2]
```diff
@@ -519,13 +519,32 @@ def get_organizer_navigation(request):
...
+        if 'can_change_organizer_settings' in request.orgapermset:
+            children.append(
+                {
+                    'label': _('Acceptance'),
+                    'url': reverse('control:organizer.giftcards.acceptance', kwargs={
+                        'organizer': request.organizer.slug
+                    }),
+                    'active': 'organizer.giftcards.acceptance' in url.url_name,
+                }
+            )
```
[the remark this answers is not in the document]
Comment (raphaelm) [reply src/pretix/control/navigation.py:540]: I don't think it's necessary, at least for now, as there's no usable configuration of it that does not involve gift cards

**Comment by pc-coholic** [comment]
> Same here. Do we really want a "half-working gift card view of foreign gift cards"? It would not provide much additional value.

True... I should have elaborated more :-) I thought about having an indiciator which organizer actually owns the giftcard and possibly omit the link to the giftcard details page if the user cannot access that organizer

**Pull request merged**